app/(afterLogin)/(timesheetInformation)/timesheet/dashboard/_components/adminPanel/AttendanceReport.tsx

```
@@ -260,4 +261,4 @@ const AttendanceReport: React.FC = () => {
   );
 };
 
-export default AttendanceReport;
+export default AttendanceReport;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)